 sse/blake2-config.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sse/blake2-config.h b/sse/blake2-config.h
index a524aa9..b6e57da 100644
--- a/sse/blake2-config.h
+++ b/sse/blake2-config.h
@@ -16,7 +16,7 @@
 #define BLAKE2_CONFIG_H
 
 /* These don't work everywhere */
-#if defined(__SSE2__) || defined(__x86_64__) || defined(__amd64__)
+#if defined(__SSE2__) || defined(__x86_64__) || defined(__amd64__) || (defined(_M_AMD64) || defined(_M_X64)) || (_M_IX86_FP == 2)
 #define HAVE_SSE2
 #endif
 
-- 
2.39.2.windows.1

